### PR TITLE
Update PandasCSVReport.ipynb

### PR DESCRIPTION
**Pull request by alvar113**
debug

### CODE REVIEW

File: Webster/PandasCSVReport.ipynb [hunk 1]
```diff
@@ -79,25 +79,22 @@
...
    "outputs": [],
    "source": [
     "# Filter data by space (sq ft)\n",
     "def filter_by_space(data, min_size, max_size):\n",
-    "        # Remove commas and convert the 'Exhibit Meeting Area' column to integers\n",
-    "    data['Exhibit Meeting Area'] = data['Exhibit Meeting Area'].str.replace(',', '').astype(int)\n",
-    "    \n",
     "    # Apply the filter without int()\n",
-    "    filtered_data = data[(data['Exhibit Meeting Area'] >= min_size) & (data['Exhibit Meeting Area'] <= max_size)]\n",
+    "    filtered_data = data[(data['Exhibit Meeting Area'].str.replace(',', '').astype(int) >= min_size) & (data['Exhibit Meeting Area'].str.replace(',', '').astype(int) <= max_size)]\n",
```
Comment (alvar113) [inline Webster/PandasCSVReport.ipynb:90]: Before you were trying to compare string to int. I added .str.replace(',', '').astype(int) to remove commas and to int so the comparison works

File: Webster/PandasCSVReport.ipynb [hunk 2]
```diff
@@ -59,26 +73,31 @@
     "        sorted_data = data.sort_values(by=\"Property Name\", ascending=False)\n",
     "    else:\n",
     "        print(\"Invalid sort order\")\n",
-    "    return data\n",
-    "    return sorted_data"
+    "    return sorted_data\n",
```
Comment (alvar113) [inline Webster/PandasCSVReport.ipynb:76]: Two returns, removed return data and kept return sorted_data

File: Webster/PandasCSVReport.ipynb (hunk 1 above)
Comment (alvar113) [reply Webster/PandasCSVReport.ipynb:90]: This fixed the filtering.

File: Webster/PandasCSVReport.ipynb (hunk 2 above)
Comment (alvar113) [reply Webster/PandasCSVReport.ipynb:76]: this fixed the sorting

File: Webster/PandasCSVReport.ipynb
Comment (webkunlv) [inline Webster/PandasCSVReport.ipynb:1]: Thanks for the assist

**Pull request merged**